remix/app/lib/handler/app.handler.ts

```diff
@@ -1,22 +1,24 @@
 import { User } from '@prisma/client';
-import { Response } from '@remix-run/node';
+import { Response, Session } from '@remix-run/node';
 import { StatusCodes } from 'http-status-codes';
 
 import { authenticator } from '../../config/auth.server';
+import { getSession } from '../../config/session.server';
 import UserRepository from '../../repositories/user.server';
 
 export default async function appHandler(
   request: Request,
-  callback: (user: User) => Promise<unknown>
+  callback: (user: User, session: Session) => Promise<unknown>
 ) {
   let userInfo: User;
 
-  const session = (await authenticator.isAuthenticated(request)) as User;
+  const userSession = (await authenticator.isAuthenticated(request)) as User;
+  const cookie = await getSession(request.headers.get('Cookie'));
```

Comment:
Super minor: I think we can move this `cookie` line to after the line:

```typescript
userInfo = await UserRepository.findBy({ email: userSession.email });
```

As we only need to use the `cookie` to pass into the `callback` funfction.

Reply:
Yep, That's sound better.

Reply:
Improved here: https://github.com/nimblehq/js-framework-benchmark/pull/120/commits/07415cea75c043a849c912f59b3c4428548ba754#diff-036147b6dd83af754aaf50a68df29df4853cc9b7360cb958764263be355efd6a